src/App.jsx

```diff
@@ -0,0 +1,28 @@
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './App.css';
```

Comment:
Follow import order (https://dev.to/otamnitram/sorting-your-imports-correctly-in-react-213m)
You don't need to change eslint config for this. `react import plugin` will add rules